plugin/evm/vm.go

```diff
@@ -990,6 +1121,7 @@ func (vm *VM) Shutdown(context.Context) error {
 	if vm.ctx == nil {
 		return nil
 	}
+	vm.cancel()
```

Comment:
Should we wait for everything that is canceled by this to finish cleaning up? Not necessary for this PR

Reply:
We already do here: https://github.com/ava-labs/coreth/blob/fbfa3726fda8ba53184cc4fef4fb82621f2328a7/plugin/evm/vm.go#L1015-L1019